### ISSUE TITLE
You can't hurt ... in their own territory unless you declare them as your enemy.

### ISSUE THREAD

**Issue by ghost** [drtshock/Factions#967]
Dear Support,

Is there any way to enable pvp in someone elses territory if you are not enemied to them?
I think this is really usefull for when you are not moderator in your  own faction and can't declare them as enemy.

I tried messing around in the config.json with these settings but without luck:
  "disablePVPBetweenNeutralFactions": false,
  "disablePVPForFactionlessPlayers": false,
  "enablePVPAgainstFactionlessInAttackersLand": true,
  "noPVPDamageToOthersForXSecondsAfterLogin": 0,
  "peacefulTerritoryDisablePVP": false,
  "peacefulTerritoryDisableMonsters": false,
  "peacefulTerritoryDisableBoom": false,
  "peacefulMembersDisablePowerLoss": false,

**Comment by MarkehMe** [comment]
We fixed this in LegacyFactions 1.5 with enforced relations.

```js
  /*
   * Relationships require both sides to set the relationship. If you want certaion relationships to automatically be enforced, set it here.
   * A better way to understand this, if you set enemy to true and either has enemy set - then enemy will be the enforced relation.
   */
  "enforcedRelations": {
    "ENEMY": false
  },
```

**Comment by Portanna** [comment]
Same problem here
